axelor-business-project/src/main/resources/views/ProjectTask.xml

```diff
@@ -81,6 +81,17 @@
     </extend>
     <extend target="//panel[@name='treeViewPanel']">
       <insert position="after">
+        <panel name="purchasePanel" title="Purchase" colSpan="12"
+          if-module="axelor-business-project" if="__config__.app.isApp('business-project')">
+          <button name="generatePurchaseOrderBtn" title="Generate Purchase Order"
+            onClick="action-project-task-generate-purchase-order"/>
+          <field name="purchaseOrderList" colSpan="12"
+            if="__config__.app.isApp('business-project')" if-module="axelor-business-project"
+            canNew="false"/>
+          <field name="purchaseOrderLineList" colSpan="12"
+            if="__config__.app.isApp('business-project')" if-module="axelor-business-project"
+            canNew="false"/>
+        </panel>
```

Comment:
- the `if-module` conditions are unnecessary here (using view extension)
- the  `if` can only be on the parent panel